packages/odie-client/src/utils/zendesk-message-converter.ts

```diff
@@ -62,18 +62,30 @@ function getContentMessage( message: ZendeskMessage ): string {
 export const zendeskMessageConverter: ( message: ZendeskMessage ) => Message = ( message ) => {
 	let type = message.type as MessageType;
 	let feedbackUrl;
+	let context: Context = { site_id: null };
+	let role = (
+		[ 'user', 'business' ].includes( message.role ) ? message.role : 'user'
+	) as MessageRole;
 
 	if ( message?.source?.type === 'zd:surveys' && message?.actions?.length ) {
 		type = 'conversation-feedback';
 		feedbackUrl = message?.actions[ 0 ].uri;
 	}
 
+	if ( message?.source?.type === 'zd:answerBot' ) {
+		type = 'message';
+		role = 'bot';
+		context = {
+			...context,
+			flags: { hide_disclaimer_content: true, show_contact_support_msg: true },
```

Comment:
We are missing the flag: `show_ai_avatar`, which was previously set as `false`. Is this intencional? I know not setting it is almost like setting it to false, but this way we are more explicit. We know it was set, rather than forgotten.

```suggestion
			flags: { hide_disclaimer_content: true, show_contact_support_msg: true, show_ai_avatar: false },
```